### DISCUSSION TITLE
Is it possible to show more than one top-level window?

### DISCUSSION THREAD

**Discussion by nathan-alden-sr** [canton7/Stylet#252] [Q&A]
I have the following code in my root view-model:
```cs
private void ShowSettingsWindow()
{
    var settingsViewModel = _lifetimeScope.Resolve<SettingsViewModel>();

    _windowManager.ShowWindow(settingsViewModel);
}
```
This displays the settings view as expected. However, it displays it _modal_, which I do not want. I have not been able to discover a way around this limitation. I want to show the window non-modal such that the user may still interact with the root view while the settings view is active.

How can I accomplish this?

**Comment by canton7** [comment]
ShowDialog displays it modal. ShowWindow should just display it non-modally. Those calls map onto `Window.ShowDialog` and `Window.Show` respectively.
